### DISCUSSION TITLE
RE 3 include the original costumes as part of Player Character options

### DISCUSSION THREAD

**Discussion by baesik** [IntelOrca/biorand#394] [Ideas]
![image](https://github.com/IntelOrca/biorand/assets/16945591/f9793372-4b62-447e-824f-1f032b0eb5ac)

**Comment by IntelOrca** [comment]
Can you explain what you mean by this?

**Reply by baesik** [reply]
For example, in the drop-down menu for it randomizing your character. You can choose Jill RE 3 but it will be her default outfit.

![BioRand_A_Resident_Evil_Randomizer_-_v3 0 205,06PM](https://github.com/IntelOrca/biorand/assets/16945591/e087d0ff-e226-410e-9c29-1ae19d5ad2ae)

Having the additional costumes added to the pool of custom NPCs, I guess is what I mean. Sorry if I'm confusing.

**Reply by IntelOrca** [reply]
Can't you just uncheck the change character checkbox, then select the costume you want in-game? I know it won't then randomize Carlos, but I could add another checkbox for that.

**Reply by baesik** [reply]
I don't think this request should qualify for its own checkbox. The point would just be to add potential player characters to the random pool. If it's not worth the trouble, the current options are fine.

**Comment by qwewqewrretrettryytuyuuyi** [comment]
Brother, would you like to add the Biohazard Veronica PSSX emulator or something? How can we play it?
